examples/notes/iOS/Notes/ContentView.swift

```diff
@@ -19,70 +12,33 @@ extension TextCursor {
     }
 }
 
-typealias Uuid = [UInt8]
-
-enum CoreEvent {
-    case event(Event)
-    // TODO: response
-}
-
-class Core: ObservableObject {
-    @Published var view: ViewModel
-
-    var num = 1
-
-    init() {
-        view = ViewModel(text: "", cursor: .position(0)) // this will get replaced by the next call
+struct ContentView: View {
+    @ObservedObject var model: Core
 
+    init(model: Core) {
+        self.model = model
         // Insert initial document for testing
-        update(event: .event(.insert("Hello from the core!")))
+        model.update(event: .insert("Hello!"))
```

Comment:
Is it worth putting update into environment, like in the tap to pay example?

Reply:
I think it's deffo worth doing when we have child components, but otherwise probs not